app/controllers/admin_controller.rb

```diff
@@ -39,6 +43,15 @@ def users
     # Filter by search text
     users = filter(users, search)
 
+    # Filter by locked boolean
+    if !locked.nil?
```

Comment:
I think this whole block can just be written as:
```
users = locked ? users.where.not(locked_at: nil) : users.where(locked_at: nil) unless locked.nil?
```

Reply:
good call, fixed here along with rubocop errors: https://github.com/SaraAlert/SaraAlert/pull/756/commits/a76bb2c3f96f884c3fe6e77f39ceb84073cce592